.gitignore

```diff
@@ -0,0 +1,18 @@
+#some of these may still be checked in; also using this
+#to ignore files when zipping repo for data release
+run_lstm_check_results.ipynb
```

Comment:
not covered by `*.ipynb` below?

Reply:
For git yes, but * isn't handled as expected for copy_zip_repo, so this is still needed it seems.